src/components/UI/TwoInputArea.js

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled';
+
+const TwoInputArea = styled.div`
+  display: flex;
+
+  div {
+    flex: 1;
+    margin-right: 25px;
+
+    &:last-child {
+      margin-right: 0px;
+    }
+  }
+`;
```

Comment:
I just thought more about this issue with the "nested" element selectors. So writing it like this, it is somewhat hard to change the styles of the inner `div`.

My suggestion is to create an additional styled component in this file like this:

```
const InnerTwoInputArea = styled.div`
  flex: 1;
  margin-right: 25px;

  &:last-child {
    margin-right: 0px;
  }
`;
```

WDYT?

Reply:
Good suggestion to make things more explicit

Reply:
ok done!